daemon/service/service_nocgo.go

```diff
@@ -0,0 +1,9 @@
+// +build !cgo
+
+// Package service implements the daemon service for mediating access to the
+// storage backend.
+package service
+
+// A dummy implementation of Serve for the environment of Cgo being disabled.
+func Serve(sockpath, dbpath string) {
```

Comment:
Please put a `select { }` to block the goroutine, thus staying close to the behaviour of the non-dummy Serve implementation. Please also add a log here (using `logger`) to make it clear that this is a dummy implementation.